okhttp/src/main/java/com/squareup/okhttp/CertificatePinner.java

```diff
@@ -180,9 +208,12 @@ private static ByteString sha1(X509Certificate x509Certificate) {
     private final Map<String, List<ByteString>> hostnameToPins = new LinkedHashMap<>();
 
     /**
-     * Pins certificates for {@code hostname}. Each pin is a SHA-1 hash of a
-     * certificate's Subject Public Key Info, base64-encoded and prefixed with
-     * "sha1/".
+     * Pins certificates for {@code hostname}.
+     *
+     * @param hostname lower-case host name or wildcard pattern such as {@code *.example.com}.
```

Comment:
Done, added comment from `OkHostnameVerifier`, I think that APIs used for "security things" should be well documented, especially `CertificatePinner` because with incorrect `hostname` definition it'll just skip check and MiTM attack will be possible, also, you can create an issue for `hostname` validation in `CertificatePinner.Builder#add()` it'll allow to prevent incorrect usage of `CertificatePinner`